functions.php

```diff
@@ -527,10 +532,75 @@ public function register_p4_page_type_taxonomy() {
 		}
 	}
 
+	/**
+	 * Save custom taxonomy for planet4 post types
+	 */
+	public function p4_save_post_type( $post_id ) {
+		// some of these checks might be redundant, but they're all nicely
```

Comment:
Minor tick: Multi-line comments are generally in the `/** */` format

Reply:
Changed to /** */ format.